src/UnitfulRecipes.jl

```diff
@@ -50,11 +50,27 @@ const AVec = AbstractVector
 const AMat{T} = AbstractArray{T,2} where T
 @recipe function f(x::AVec, y::AVec, z::AMat{T}) where T <: Quantity
     u = get(plotattributes, :zunit, unit(eltype(z)))
+    ustripattribute!(plotattributes, :clims, u)
     z = fixaxis!(plotattributes, z, :z)
     append_unit_if_needed!(plotattributes, :colorbar_title, u)
     x, y, z
 end
 
+# Recipe for heatmap(z), surface(z)
+# Caution: also gets use for plot(z::Matrix)
+@recipe function f(z::AMat{T}) where T <: Quantity
+    u = get(plotattributes, :zunit, unit(eltype(z)))
+    if get(plotattributes, :seriestype, :nothing) ∈ (:heatmap, :surface)
```

Comment:
I think also `contour`? See also
https://github.com/rafaqz/DimensionalData.jl/blob/c0bd6ae45a485fc3b5fdbc68a209666d9425110c/src/plotrecipes.jl#L22
but I don't think they all have a colorbar. We can also start with just few heatmap like plots and extend the list, as issues arise.

Reply:
I added `contour` and `contourf` (which was not in that list).
I checked out a few of the others that I could figure out how to use and did not see any other colorbars.
I agree with waiting for issues.

Reply:
I just wasn't sure if a `moar` plot was a type of plot that I didn't know about, because there is also this closely related word that seems like it could be a type of plot https://en.wikipedia.org/wiki/Mohair

Reply:
https://www.urbandictionary.com/define.php?term=moar